README.md

```diff
@@ -1 +1,183 @@
 # cypress-selector-generator
+
+Tired of the boilerplate needed for writing unambiguous selectors for Cypress UI tests? Use `cypress-selector-generator` to create navigation shorthands!
+
+With this plugin, the following examples have full IntelliSense, and you can chain arbitrary Cypress commands off of your app selectors:
+```js
+myApp.appInfo.createNewUserSection.firstNameInput
+    .type('First');
+myApp.appInfo.createNewUserSection.lastNameInput
+    .type('Last');
+myApp.appInfo.createNewUserSection.submitButton
+    .click();
+```
+```js
+myApp.appInfo.createNewUserSection
+    .within(({ firstNameInput, lastNameInput, submitButton }) => {
+        firstNameInput
+            .type('First');
+        lastNameInput
+            .type('Last');
+        submitButton
+            .click();
+    });
+```
+
+The above snippets equivalent to the following raw Cypress:
+```js
+cy.get('[data-test=app_info] [data-test=create_new_user_section] [data-test=first_name_input]')
+    .type('First');
+cy.get('[data-test=app_info] [data-test=create_new_user_section] [data-test=last_name_input]')
+    .type('Last');
+cy.get('[data-test=app_info] [data-test=create_new_user_section] [data-test=submit_button]')
+    .click();
+```
+```js
+cy.get('[data-test=app_info] [data-test=create_new_user_section]')
+    .within(() => {
+        cy.get('[data-test=first_name_input]')
+            .type('First');
+        cy.get('[data-test=last_name_input]')
+            .type('Last');
+        cy.get('[data-test=submit_button]')
+            .click();
+    });
+```
+
+Or, if you would prefer sticking closer to normal Cypress syntax, there's a new `cy.tget` command. It works the same way as the built-in `cy.get`, but automatically wraps non-css/jQuery identifiers with `[data-test]`.
+
+For example: 
+```js
+cy.tget('app_info:visible create_new_user_section submit_button');
+```
+will be converted automatically into 
+```js
+cy.get('[data-test=app_info]:visible [data-test=create_new_user_section] [data-test=submit_button]');
+```
+
+## Installation
+```bash
+pnpm add -D cypress-selector-shorthand
+# or
+npm i -D cypress-selector-shorthandgenerator
+# or
+yarn add -D cypress-selector-shorthand
+```
+
+Add this line to your project's `cypress/support/commands.js` (or `commands.ts`):
+```js
+import 'cypress-selector-shorthand/install';
+```
+
+## Configuration
+To make use of the navigation shorthands, there's three extra steps you need to do to generate the objects and interfaces. (Note: you can use `cy.tget()` without this.)
+
+First, you need to create a schema JSON file, e.g. the following might be used for the example at the top of this file:
```

Comment:
Would a typical Cypress dev know where to place the schema file?

Reply:
Location can be anywhere, since it is specified by `--infile` and has no default